pandas/tests/window/test_apply.py

```diff
@@ -52,8 +52,7 @@ def test_rolling_apply_out_of_bounds(engine_and_raw):
 
 
 @pytest.mark.parametrize("window", [2, "2s"])
-@pytest.mark.parametrize("step", [None])
-def test_rolling_apply_with_pandas_objects(window, step):
+def test_rolling_apply_with_pandas_objects(window):
```

Comment:
i would be ok with adding a step  with [None, 1] which should just work on the numeric window ones

Reply:
great